index.js

```diff
@@ -158,21 +166,17 @@ module.exports = class Filter extends Plugin {
   }
 
   async build() {
-    // @ts-ignore
-    let srcDir = this.inputPaths[0];
-    // @ts-ignore
-    let destDir = this.outputPath;
 
     if (this.dependencyInvalidation && !this.dependencies) {
-      this.dependencies = this.processor.initialDependencies(srcDir);
+      this.dependencies = this.processor.initialDependencies(this.inputPaths);
```

Comment:
Passing `this.inputPaths` here breaks the encapsulation we hope to to achieve with `this.input` `this.output`, is this needed or can the Dependency code be made aware of `this.input`

Reply:
^^